scripts/make_release_notes.py

```diff
@@ -201,13 +209,13 @@ def parse_url(self, m):
     return self.renderer.url(m.group(1))
 
   local_issue_link = re.compile(
-      r'\('           # opening paren
-      r'#(\d+)'       # hash and issue number
-      r'\)'           # closing paren
+      r'\(?'             # opening paren
```

Comment:
The opening paren shouldn't be optional, should it?

Basically: if you're willing to match `#issue-number` without the paren then we don't need it in the pattern at all.

Also, if you're willing to just match `#issue-number` then you can simplify this by just matching the `#(\d+)\b` without even matching the list. Treat them each independently as a single issue link to replace.

Reply:
You're right. Fixed a similar issue for the closing paren as well.